README.md

```diff
@@ -72,7 +72,7 @@ foreach (var stat in varusRanked.Stats)
 }
 ```
 
-You can find a list of all the available operations in [IRiotApi](RiotSharp/IRiotApi.cs).
+You can find a list of all the available operations in [TournamentApi in Documentation](http://benfradet.github.io/RiotSharp/api/RiotSharp.RiotApi.html).
```

Comment:
As far as my English goes, I would just drop `TournamentApi `and make it like this:
`You can find a list of all the available operations in the [documentation] (...).`

This sounds more fluent for me, but I'm not a native speaker.